plugins/modules/source_control/gitlab/gitlab_group_members.py

```diff
@@ -165,98 +201,187 @@ def main():
     argument_spec.update(dict(
         api_token=dict(type='str', required=True, no_log=True),
         gitlab_group=dict(type='str', required=True),
-        gitlab_user=dict(type='str', required=True),
+        gitlab_user=dict(type='list', elements='str'),
         state=dict(type='str', default='present', choices=['present', 'absent']),
-        access_level=dict(type='str', required=False, choices=['guest', 'reporter', 'developer', 'maintainer', 'owner'])
+        access_level=dict(type='str', choices=['guest', 'reporter', 'developer', 'maintainer', 'owner']),
+        purge_users=dict(type='list', elements='str', choices=['guest', 'reporter', 'developer', 'maintainer', 'owner']),
+        gitlab_users_access=dict(
+            type='list',
+            elements='dict',
+            options=dict(
+                name=dict(type='str'),
+                access_level=dict(type='str', choices=['guest', 'reporter', 'developer', 'maintainer', 'owner']),
+            )
+        ),
     ))
 
     module = AnsibleModule(
         argument_spec=argument_spec,
         mutually_exclusive=[
             ['api_username', 'api_token'],
             ['api_password', 'api_token'],
+            ['gitlab_user', 'gitlab_users_access'],
+            ['access_level', 'gitlab_users_access'],
         ],
         required_together=[
             ['api_username', 'api_password'],
+            ['gitlab_user', 'access_level'],
         ],
         required_one_of=[
-            ['api_username', 'api_token'],
-        ],
-        required_if=[
-            ['state', 'present', ['access_level']],
+            ['gitlab_user', 'gitlab_users_access'],
         ],
+        # TODO: this is only true if gitlab_user and not gitlab_users_access is used
+        #        required_if=[
+        #            ['state', 'present', ['access_level']],
```

Comment:
```suggestion
            ['state', 'present', ['access_level', 'gitlab_users_access'], True],
```
This makes sure that if `state=present`, at least one of `access_level` and `gitlab_users_access` is present.

Reply:
> This makes sure that if `state=present`, at least one of `access_level` and `gitlab_users_access` is present.

Thank you, didn't know how to write this condition down.